Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewReadModel.cs

```diff
@@ -14,6 +14,32 @@ public ItSystemUsageOverviewReadModel()
             ItSystemTaskRefs = new List<ItSystemUsageOverviewTaskRefReadModel>();
             SensitiveDataLevels = new List<ItSystemUsageOverviewSensitiveDataLevelReadModel>();
             ItProjects = new List<ItSystemUsageOverviewItProjectReadModel>();
+            ArchivePeriods = new List<ItSystemUsageOverviewArchivePeriodReadModel>();
+        }
+
+
+        public DateTime? ActiveArchivePeriodEndDate
+        {
+            get
+            {
+                ItSystemUsageOverviewArchivePeriodReadModel archivePeriodWithEarliestStartDate = null;
+                var today = DateTime.Now;
+                foreach (var archivePeriod in ArchivePeriods)
+                {
+                    if (today >= archivePeriod.StartDate && today <= archivePeriod.EndDate)
+                    {
+                        if (archivePeriodWithEarliestStartDate == null)
+                        {
+                            archivePeriodWithEarliestStartDate = archivePeriod;
+                        }
+                        else if (archivePeriodWithEarliestStartDate?.StartDate > archivePeriod.StartDate)
```

Comment:
Det er en skør logik. Man skulle tro at det handlede om hvilken en af de to aktive perioder der sluttede først men jeg kan godt se at logikken er som den var